### DISCUSSION TITLE
Just a short note of great appreciation

### DISCUSSION THREAD

**Discussion by anton-essentiapps** [trevordevore/levure#182] [General]
Hi gang! I just felt the strong urge to pop by and show my appreciation to Trevor, for this insane Levure Framework and all the other immensely helpful life savers he has contributed.

I'm in my 6th week with Levure and so far it has not only changed my LC development life for the better, it has REVOLUTIONIZED every aspect of it. Everything works like a charm. The interplay of the LC + Sublime Text + Git + LC Server + Levure quintet is absolutely phantastic and extremely stable. I wonder how I could live without it for so long.

I'm in the process of developing a quite large project and have alomost finished with porting every component of it into a Levure friendly structure. Now, my quite complex app works more stable than ever and with a much cleaner structure and workflow. How damn cool!!!

So far, the only issue I'm facing is when have to I turn off Messages in the LC IDE for and turn it on again, LC starts throwing Levure related errors and any access to Levure functions seems to be broken. I haven't dug deeper into researching the causes and possible remedies yet, but when I evertually do, I'll let you know.

Please dear Trevor, don't quit on maintaining and improving this great framework! A great, GREAT thank you!!

**Comment by trevordevore** [comment]
You're welcome @Tonaras. I'm glad you enjoy using it!

I haven't been spending as much time coding in LiveCode recently due to my other responsibilities in my business. The beauty of having the project on Github is that anyone can study and contribute to the code.
